j_core/businessobject/BusinessObject.py

```diff
@@ -17,6 +19,9 @@ class BusinessObject:
     """ Must be extended in conjunction with Base """
     id: Mapped[int] = mapped_column(primary_key=True)
 
+    def __init__(self, *args: Any, **kwargs: Any):
```

Comment:
This empty constructor only serves to silence type hints. When working with the `Registry`in `businessobject` and searching for a specifict `Type[BusinessObject` to instantiate it, the type hint would not see the default constructor, otherwise.

Reply:
I did add a comment to explain why it's there.